config/default.go

```diff
@@ -51,6 +51,7 @@ var defaultConfig = &Config{
 			ServiceStatus: []string{"passing"},
 			CheckInterval: time.Second,
 			CheckTimeout:  3 * time.Second,
+			CheckScheme:   "http",
```

Comment:
I'm thinking that you could default to `"" == "http"`. Would save you an `else`. What do you think?